docs/development/IntegrationTest.md

```diff
@@ -0,0 +1,62 @@
+<!---      
+  Licensed under the Apache License, Version 2.0 (the "License");      
+  you may not use this file except in compliance with the License.      
+  You may obtain a copy of the License at      
+      
+   http://www.apache.org/licenses/LICENSE-2.0      
+      
+  Unless required by applicable law or agreed to in writing, software      
+  distributed under the License is distributed on an "AS IS" BASIS,      
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.      
+  See the License for the specific language governing permissions and      
+  limitations under the License. See accompanying LICENSE file.      
+-->
+
+# IntegrationTest
+
+## Run the existing tests.
+##### Move to the working directroy.
+```
+cd hadoop-submarine/submarine-test/e2e
```

Comment:
Need change `hadoop-submarine` to `submarine`.

Reply:
> Need change `hadoop-submarine` to `submarine`.

@LinhaoZhu ,
I have changed above two things. Thank you.